tox.ini

```diff
@@ -13,7 +13,7 @@ python =
     3.11: py311
 
 [testenv]
-passenv = CI GITHUB_WORKFLOW
+passenv = CI,GITHUB_WORKFLOW
```

Comment:
I totally missed this change from `tox` 3 to 4, thank you very much!
BTW, `tox` is not used anymore in the `CI`, for this reason I have not noticed the breaking change.

Reply:
It’s used for the lint checks!  (see preceding commit)

Reply:
`tox` is still used but these env variables are not used anymore.

Reply:
Ah I understand!  tox still complained about the format (no spaces allowed)